infra/gcp/bash/ensure-staging-gcb-untrusted.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates & configures the project used for running untrusted cloudbuild jobs.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0" > /dev/stderr
+    echo > /dev/stderr
+}
+
+if [ $# != 0 ]; then
+    usage
+    exit 1
+fi
+
+# The GCP project name.
+PROJECT=$(k8s_infra_project "staging" "k8s-staging-gcb-untrusted")
+
+# The service account name in $PROJECT.
+GCB_SVCACCT="builder"
+
+
+color 6 "Ensuring project exists: ${PROJECT}"
+ensure_project "${PROJECT}"
+
+GCB_PROJECT_SERVICES=(
+    cloudbuild.googleapis.com
+    artifactregistry.googleapis.com
```

Comment:
Why do we need this service ? are we intending to push the images build at presubmti ?

Reply:
No. Instructions from Google.

https://cloud.google.com/build/docs/build-push-docker-image

Reply:
But instructions is about build+push. we are stopping at the build step. We should not able this service just to satisfy doc guidance.